code/go/0chain.net/blobber/main.go

```diff
@@ -469,8 +468,8 @@ func SetupBlobberOnBC(logDir string) {
 	var logName = logDir + "/0chainBlobber.log"
 	zcncore.SetLogFile(logName, false)
 	zcncore.SetLogLevel(3)
-	zcncore.InitZCNSDK(serverChain.BlockWorker, config.Configuration.SignatureScheme)
-	zcncore.SetWalletInfo(node.Self.GetWalletString(), false)
+	_ = zcncore.InitZCNSDK(serverChain.BlockWorker, config.Configuration.SignatureScheme)
```

Comment:
unchecked and it will fail later on if `InitZCNSDK` throws an error, but perhaps it might be better to fail early?